examples/pxScene2d/src/pxShaderResource.cpp

```diff
@@ -575,6 +587,104 @@ void pxShaderResource::loadResourceFromArchive(rtObjectRef /*archiveRef*/)
 #endif // 0
 }
 
+void pxShaderResource::processDownloadedResource(rtFileDownloadRequest* fileDownloadRequest)
+{
```

Comment:
`pxShaderResource::processDownloadedResource()` is an exact copy of the base class version.  Let's remove this version to allow for easy code maintenance.

Reply:
Fixed a callback bug in base class.   The removes the need for pxShaderResource::processDownloadedResource() in my version,